tests/any_validator_test.js

```diff
@@ -132,4 +132,14 @@ describe('Duns - Any Validator', function() {
     should(Duns.validate('10', schema)).be.false;
   });
 
+  it('Can format value', function() {
+    var schema = Duns.any().returns(function(val) {
+      return val * 2;
+    });
+
+    should(schema.init(10).format(10)).be.eql(20);
```

Comment:
Perhaps the init value should be changed to for example 5 or something. To illustrate that the `.format` has precedence.